p22/Cargo.toml

```diff
@@ -3,7 +3,6 @@ name = "p22"
 version = "0.1.0"
 edition = "2024"
 
-#[dependencies]
-
-[dev-dependencies]
-rand = "0.8" 
\ No newline at end of file
+[[bin]]
+name = "twelve_days"
+path = "src/bin/twelve_days.rs"
```

Comment:
You don't need to explicitly list this binary. Files in the src/bin folder are automatically interpreted as binaries.